src/WhatToCook.Application/Domain/PlanOfmeals.cs

```diff
@@ -2,9 +2,10 @@
 
 public class PlanOfMeals
```

Comment:
Fix casing in filename and add constructor

Reply:
fixed, will be in the next commit